src/lib/transition_frontier/full_frontier/full_frontier.ml

```diff
@@ -440,8 +446,12 @@ let move_root t ~new_root_hash ~new_root_protocol_states ~garbage
   (* rewrite the new root breadcrumb to contain the new root mask *)
   let new_root_breadcrumb =
     Breadcrumb.create
-      (Breadcrumb.validated_transition new_root_node.breadcrumb)
-      new_staged_ledger
+      ~validated_transition:
+        (Breadcrumb.validated_transition new_root_node.breadcrumb)
+      ~staged_ledger:
+        new_staged_ledger
+        (* TODO: ditto above; seems like this should carry over *)
+      ~just_emitted_a_proof:false
```

Comment:
I think this (and the other uses of `Breadcrumb.create`) is ok because we only ever check this field in the new root breadcrumb. The new root breadcrumb will always have the correct value because it would have been generated using `Breadcrumb.build`. But maybe, for completeness, we should just persist `just_emitted_a_proof` as well?